### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by MCWertGaming**
This PR adds PR templates.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**Optionally: Add additional Things to this PR, or a picture of a furry animal :3**
+
+**TODO**
+
+<!--Add things here that still need to be done in this PR.-->
+
+- [ ] Applied `npm run format`
+- [ ] Tested the changes locally
+
```
Comment (Sigmanificient) [inline .github/pull_request_template.md:24]:
```suggestion
- [ ] Added breaking changes
```

[the remark this answers is not in the document]
Comment (MCWertGaming) [reply .github/pull_request_template.md:24]: what does the build mode change?

**Comment by MCWertGaming** [comment]
Done @Sigmanificient.

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sigmanificient) [reply .github/pull_request_template.md]: Well local build is something quite fast to see what you implement, it also have some more stuff like error report, etc
Production build try to remove unused stuff & optimized to get a lightway and fast buid. With the frontend Tooling tool they can be some changes like assets being hashed for example. Already had a bug with this

**Pull request merged**